go/vumitools/account/models.py

```diff
@@ -29,11 +29,19 @@ class UserAppPermission(Model):
 class UserAccount(Model):
     """A user account."""
 
-    VERSION = 4
+    VERSION = 6
     MIGRATOR = UserAccountMigrator
 
+    ACCOUNT_TYPE_NORMAL = u'normal'
+    ACCOUNT_TYPE_DEVELOPER = u'developer'
```

Comment:
I think billing is associated with a campaign / project rather than an account. Since our user accounts are really campaigns currently, I think maybe this is the correct place?